scheduled-jobs-alert.tf

```diff
@@ -111,3 +111,38 @@ module "delete-rejected-files-alert" {
   trigger_threshold          = 0
   resourcegroup_name         = "${azurerm_resource_group.rg.name}"
 }
+
+module "check-new-envelopes-alert" {
+  source            = "git@github.com:hmcts/cnp-module-metric-alert"
+  location          = "${azurerm_application_insights.appinsights.location}"
+  app_insights_name = "${azurerm_application_insights.appinsights.name}"
+
+  enabled    = "${var.env == "prod"}"
+  alert_name = "Check-New-Envelopes"
+  alert_desc = "Triggers when no logs from check-new-envelopes job found within timeframe."
+
+  app_insights_query =  <<EOF
+traces // The scheduled job which runs on business hours only
+| where message startswith "Started check-new-envelopes job"
+| union (datatable(metric: real) [
+  0, // to mimic the non business hours
+]
+| extend timeNow = now()
+| extend day_of_week = toint(substring(tostring(dayofweek(timeNow)), 0, 1))
+| extend hour_of_the_day = datetime_part("hour", timeNow)
+| where day_of_week == 0 or // sunday
+        day_of_week == 6 or // saturday
+        hour_of_the_day < 9 or // scheduled job start hour
+        hour_of_the_day  > 17) // scheduled job end hour
```

Comment:
from acceptance criteria:
>If no envelopes are found then log with an **appropriate message**. (Check BSP)
>Define a new alert that would be triggered **if the message is present**

Can we simply send an email if a warning is logged by the job?

Reply:
ie:
```
traces
| where message startswith "No Envelopes created"
```
```

Reply:
The alerts for this is already created in a separate PR 
https://github.com/hmcts/reform-scan-shared-infra/pull/34.
This PR is to make sure the scheduled job (https://github.com/hmcts/blob-router-service/pull/341) is running in business hours (the cron for the scheduled job is in https://github.com/hmcts/blob-router-service/pull/341)

Reply:
ok, I got confused by the PR title.